domains/jimmster.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "A unsusually stupid developers programming blog,
```

Seems that your string isn't closed off at the end.
```suggestion
  "description": "A unsusually stupid developers programming blog,"
```